src/Dapplo.Jira.Tests/IssueTests.cs

```diff
@@ -65,6 +98,45 @@ public async Task CreateIssue()
 			// Remove again
 			await Client.Issue.DeleteAsync(createdIssue.Key);
 		}
+		
+		[Fact]
+		public async Task CreateIssueWithCustomFields()
+		{
+			var meMyselfAndI = await Client.User.GetMyselfAsync();
+			Assert.NotNull(meMyselfAndI);
+
+			var issueTypes = await Client.Issue.GetIssueTypesAsync();
+			var projects = await Client.Project.GetAllAsync();
+
+			var bugIssueType = issueTypes.First(type => type.Name == "Bug");
+			var projectForIssue = projects.First(digest => digest.Key == "DIT");
+
+			var cfTextField = "customfield_10001";
+			var cfLabelField = "customfield_10002";
```

Comment:
FYI
The tests broke with a message:
`Field 'customfield_10001' cannot be set. It is not on the appropriate screen, or unknown.`
I'm guessing these are not available in my JIRA, didn't have time to go though all of it.